htdocs/societe/list.php

```diff
@@ -153,7 +153,7 @@
 if (($tmp = $langs->transnoentities("ProfId6".$mysoc->country_code)) && $tmp != "ProfId6".$mysoc->country_code && $tmp != '-') $fieldstosearchall['s.idprof6'] = 'ProfId6';
 if (!empty($conf->barcode->enabled)) $fieldstosearchall['s.barcode'] = 'Gencod';
 // Personalized search criterias. Example: $conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS = 's.nom=ThirdPartyName;s.name_alias=AliasNameShort;s.code_client=CustomerCode'
-if (!empty($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS)) $fieldstosearchall = dolExplodeIntoArray($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS);
+if (!empty($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS)) $fieldstosearchall += dolExplodeIntoArray($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS);
```

Comment:
Goal of option THIRDPARTY_QUICKSEARCH_ON_FIELDS is to be able to decide which param is used for quick search. Making this change allows only to complete, that is not enough.

Reply:
@eldy 
its not the comportement in V11

Reply:
in v11, the code is 
if (!empty($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS)) $fieldstosearchall = dolExplodeIntoArray($conf->global->THIRDPARTY_QUICKSEARCH_ON_FIELDS);

So it means the fieldstosearch array is completely replaced with the expected values in the setup. So in v11 we are able to decide which field is used or not. This is the expected behavior.

Reply:
Ok so PR close